scrapi/harvesters/lwbin.py

```diff
@@ -0,0 +1,210 @@
+            authors.extend(split_name)
+
+    contributor_list = []
+    append_emails = True
+    if len(authors) != 1 or len(emails) != 1 or emails[0] == u'':
+        append_emails = False  # append the email to the author only when 1 record is observed
+
+    for ind, author in enumerate(authors):
+        if is_organization(author):
+            contributor = {
+                'name': author,
+                'sameAs': []
+            }
+        else:
+            name = HumanName(author)
+            contributor = {
+                'name': author,
+                'givenName': name.first,
+                'additionalName': name.middle,
+                'familyName': name.last,
+                'sameAs': []
+            }
+        if append_emails:
+            contributor['email'] = emails[ind]
+        contributor_list.append(contributor)
+
+    if not append_emails and emails[0] != u'':
+        for email in emails:
+            contributor = {
+                'name': '',
+                'givenName': '',
+                'additionalName': '',
+                'familyName': '',
+                'email': email,
+                'sameAs': [],
+            }
+            contributor_list.append(contributor)
+
+    return contributor_list
+
+
+def process_licenses(license_title, license_url, license_id):
+    """Process licenses to comply with the noormalized schema
+    """
+
+    if not license_url:
+        return []
+    else:
+        license = {
+            'uri': license_url,
+            'description': "{} ({})".format(license_title, license_id) or ""
+        }
+
+        return [license]
+
+
+def construct_url(url, dataset_path, end_point):
+    """Return a url that directs back to the page on LBWIN Data Hub instead of the source page.
+
+    Keyword arguments:
+    url -- host url
+    dataset_path -- parent path of all datasets
+    end_point -- name of datasets
+    """
+
+    return "/".join([url, dataset_path, end_point])
+
+
+def process_object_uris(url, extras):
+    """Extract doi from /extras, and return a list or object uris including /url and doi if it exists.
+    """
+    doi = ""
+    for d in extras:
+        if d['key'] == "DOI" or d['key'] == "DOI:":
+            doi = d['value']
+            break
+    if doi == "":
+        return [url]
+    else:
+        return [url, doi]
+
+
+class LWBINHarvester(JSONHarvester):
+    short_name = 'lwbin'
+    long_name = 'Lake Winnipeg Basin Information Network'
+    url = 'http://130.179.67.140'
+    dataset_path = "dataset"   # dataset base url for constructing urls that go back to LWBIN instead of source pages.
+
+    DEFAULT_ENCODING = 'UTF-8'
+
+    record_encoding = None
+
+    @property
+    def schema(self):
+        return {
+            'title': ('/title', lambda x: x if x else ''),
+            'description': ('/notes', lambda x: x[0] if (isinstance(x, list) and x) else x or ''),
```

Comment:
I think we have a helper function for this (`scrapi.base.helpers.single_result`)
```python
'description': ('/notes', compose(lambda x: single_result(x, x) or ''))
```